led-display/client.ts

```diff
@@ -212,13 +222,13 @@ namespace modules {
         //% group="LED Display"
         setPixelColor(index: number, rgb: number) {
             index = index | 0
-            const pixels = this.pixels()
+            const pixels = this._localPixels;
+            if (!pixels) return
             if (index >= 0 && (index + 1) * 3 <= pixels.length) {
                 pixels[index * 3] = (rgb >> 16) & 0xff
                 pixels[index * 3 + 1] = (rgb >> 8) & 0xff
                 pixels[index * 3 + 2] = rgb & 0xff
             }
-            this.setPixels(pixels)
```

Comment:
when do you decide to send the buffer to the device?

Reply:
it's up to the user to call show().